modules/services/redshift-gammastep/lib/options.nix

```diff
@@ -26,13 +26,31 @@ in {
   in [
     (mkRemovedOptionModule [ "services" moduleName "extraOptions" ]
       "All ${programName} configuration is now available through services.${moduleName}.settings instead.")
-    (mkRenamed [ "brightness" "day" ] "brightness-day")
-    (mkRenamed [ "brightness" "night" ] "brightness-night")
```

Comment:
I would prefer to keep `mkRenamed` and fixing them, they should work and need to figure out why it doesn't.

Reply:
Also, the reason that I would prefer to not re-added them is because there is no reason this option is more important them the others in this same module (like `gamma`), and they don't have any special attribute either.

Reply:
But if you're going for this route remove `mkRenamed` function.

Reply:
Yea, not sure why `mkRenamed` isn't working as expected in this case. It looks to be the proper usage to me.

Reply:
Found the issue, opening an alternative PR.

Reply:
Let me know if it's in the module or the function itself. You might need to update the similar `nixpkgs` PR you have open if it's in the module.

Once complete I'll close this PR since it probably won't be necessary.

Reply:
https://github.com/nix-community/home-manager/pull/1804

> You might need to update the similar nixpkgs PR you have open if it's in the module.

Sure. I don't know if I will keep pushing this PR in nixpkgs though, since its efforts stalled.